### PR TITLE
Script to migrate old OPs from MySQL to Postgres

### PR DESCRIPTION
**Pull request by machisuji**

### CODE REVIEW

File: docs/operations/migrating/docker/postgresql-migration.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Migrating your Docker OpenProject database to PostgreSQL
 
-This guide will migrate your docker-based MySQL installation to a PostgreSQL installation using [pgloader](https://github.com/dimitri/pgloader).
```
Comment (oliverguenther) [inline docs/operations/migrating/docker/postgresql-migration.md:3]: Please move this somewhere to https://github.com/opf/openproject/tree/dev/help/installation-and-operations so the new help picks it up. The docs folder will be removed

**Review by oliverguenther — CHANGES_REQUESTED** [review]
Script is looking good to me. Please change the documentation place as commented

File: docs/operations/migrating/docker/postgresql-migration.md (hunk 1 above)
Comment (crohr) [reply docs/operations/migrating/docker/postgresql-migration.md:3]: Should be put within https://github.com/opf/openproject/tree/dev/help/installation-and-operations/misc

**Comment by oliverguenther** [comment]
I'll move the docs myself as part of https://community.openproject.com/wp/32154

**Pull request merged**